sdk/python/lib/pulumi/automation/_project_settings.py

```diff
@@ -186,6 +186,8 @@ def __init__(
             "python",
             "go",
             "dotnet",
+            "yaml",
+            "java"
         ]:
             raise ValueError(
                 f"Invalid value {runtime!r} for runtime. "
```

Comment:
The string below here should be updated as well. 

 But also more generally, should we even validate this at all?  After all third party language plugins can exist (and do at least for scala afair)

Reply:
I would like an API to call in Neo to check if a folder is a Pulumi project. Which do you suggest to use?